pallets/anmol-nft/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "anmol-nft"
+description = "NFT pallet for extended functionality"
+repository = "https://github.com/open-web3-stack/open-runtime-module-library/tree/master/nft"
+license = "Apache-2.0"
+version = "0.1.0"
```

Comment:
The original pallet was at `0.4.0`, I didn't think that this made sense to keep and so decided to start it off at a lower version.

Reply:
See if description needs to be updated after this: https://github.com/anmolnetwork/anmol-node/pull/32#discussion_r657830398

Reply:
I think so. I went for `A pallet to extend for building advanced NFT functions`